style.css

```diff
@@ -527,10 +741,86 @@ ul {
     flex-direction: row;
   }
 
+  #reverse {
+    flex-direction: row-reverse;
+  }
+
   .work-section {
-    grid-template-rows: repeat(4, auto);
-    width: 83.5%;
-    margin: 0 auto;
+    width: fit-content;
+    height: max-content;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    gap: 24px;
+    padding-top: 24px;
+  }
+
+  .work-header {
+    height: auto;
+    padding: 24px;
+    gap: 24px;
+  }
+
+  .img-mobile {
+    display: none;
+  }
+
+  .image-desktop {
+    display: block;
+  }
+
+  .popupDisplay {
+    width: fit-content;
+    height: 100%;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin: 1px 70px;
+    left: 1px;
+    overflow-y: scroll;
+  }
+
+  .headerDisplay {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+  }
+
+  .project-head {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 32px;
+    line-height: 44px;
+  }
+
+  .positionP {
+    display: flex;
+    align-items: baseline;
+    flex-direction: row;
+  }
+
+  .proDescription {
+    width: 50%;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+  }
+
+  .imgP {
+    width: 80%;
+    height: 70%;
+  }
+
+  .closeButton-class {
+    padding: 12px;
+    background-color: #fff;
+    background-image: url(./images/closeIcon.svg);
+    background-repeat: no-repeat;
+    float: right;
+    align-self: flex-end;
```

Comment:
- [ ] [OPTIONAL] Please remove the border around the `x-icon` 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/180295281-570c0b93-b9a8-4ea7-81f6-f391428965a3.png)  |  ![image](https://user-images.githubusercontent.com/74814780/180295310-0e392cca-fccc-4ffd-bcab-696ded607c32.png)